ui/src/app/job-details/resources/resources.component.ts

```diff
@@ -27,13 +27,15 @@ export class JobResourcesComponent implements OnInit {
   sourceFile: string = '';
   inputs: Array<string> = [];
   outputs: Array<string> = [];
+  eventDetails: Array<object> = [];
```

Comment:
Should use the actual Typescript model type, rather than `object`

Reply:
What is the best practice to add a model type? Creating a new file or another way?

Reply:
You already generated one in this PR via swagger: see `Event.ts`